src/main.ts

```diff
@@ -62,6 +75,12 @@ const command: Command = {
 			default: false
 		});
 
+		options('i', {
```

Comment:
locally supports the `SeleniumTunnel`. `saucelabs`, `browserstack` and `testingbot` support (in their equivalent format):

```ts
{ browserName: 'internet explorer', version: [ '11' ], platform: 'WIN8' },
{ browserName: 'firefox', version: '49', platform: 'WIN10' },
{ browserName: 'chrome', platform: 'WIN10' }
```

What I meant be supporting an option for `target` or `environment` was to rename the existing `config` option (as it's a confusing name, the consumer does not know that `saucelabs` actually maps back to a configuration).

There are already configuration options for `username` and `key` that configure the tunnel.

The `.dojorc` is really to configure our commands or for use internally by the CLI and it's commands not to configure the configuration for the underlying tool, so for me it is use the existing pre-configured commands or `eject`.